### PR TITLE
Bump ABI version for C# AOT

### PR DESCRIPTION
**Pull request by RReverser**
# Description of Changes

Looks like these constants got missed during upgrade, making AOT version ABI-incompatible.

Unfortunately, there's no way to use a central constant here, and we don't have the Windows + Emscripten setup for testing the AOT variant yet, so we probably should add it to some "steps to do during ABI version bump" doc.

# API and ABI breaking changes

If this is an API or ABI breaking change, please apply the
corresponding GitHub label.

# Expected complexity level and risk

*How complicated do you think these changes are? Grade on a scale from 1 to 5,
where 1 is a trivial change, and 5 is a deep-reaching and complex change.*

*This complexity rating applies not only to the complexity apparent in the diff,
but also to its interactions with existing and future code.*

*If you answered more than a 2, explain what is complex about the PR,
and what other components it interacts with in potentially concerning ways.*

# Testing

*Describe any testing you've done, and any testing you'd like your reviewers to do,
so that you're confident that all the changes work as expected!*

- [x] *Write a test you've completed here.*
- [ ] *Write a test you want a reviewer to do here, so they can check it off when they're satisfied.*

### CODE REVIEW

**Review by kazimuth — APPROVED** [review]
It would be nice to get an automated test harness up to catch these things, but that is going to be a huge pain.

I think github actions technically can use windows runners... you probably have to write PowerShell though. I've heard PowerShell is good, but never really learned it...

File: crates/bindings-csharp/Runtime/RawBindings.cs [hunk 1]
```diff
@@ -63,28 +62,36 @@ public class StdbException : Exception
 
     public class NoSuchTableException : StdbException
     {
-        internal NoSuchTableException() : base(ERRNO_NO_SUCH_TABLE) { }
+        internal NoSuchTableException()
```
Comment (kazimuth) [inline crates/bindings-csharp/Runtime/RawBindings.cs:65]: How come this formatting change ended up in this PR? Did the autoformatter in use change or something?

Comment (RReverser) [reply crates/bindings-csharp/Runtime/RawBindings.cs:65]: I think @coolreader18 just didn't have autoformatted configured in their editor, but I do, so it formatted code on save.

**Comment by RReverser** [comment]
> It would be nice to get an automated test harness up to catch these things, but that is going to be a huge pain.

TBH we could just do a search for `spacetime_7.0` (substitute current version) in the codebase. I'm intentionally keeping those strings raw (not interpolated) to make search-replace easier, so in theory we could do a dumb search in CI as well.

**Pull request merged**